core/internal/cltest/cltest.go

```diff
@@ -44,7 +44,7 @@ import (
 
 	"github.com/smartcontractkit/chainlink-relay/pkg/loop"
 
-	clienttypes "github.com/smartcontractkit/chainlink/v2/common/chains/client"
+	commonclient "github.com/smartcontractkit/chainlink/v2/common/client"
```

Comment:
Is `client` available?
```suggestion
	"github.com/smartcontractkit/chainlink/v2/common/client"
```